repo2docker/app.py

```diff
@@ -272,6 +296,15 @@ def initialize(self):
             self.run = False
             self.push = False
 
+        if args.volumes and not args.run:
+            # Can't mount if we aren't running
+            print("Can not mount volumes when not running container")
```

Comment:
can we turn this into `To mount ("-v") directories you need to also run the container ("--run").` or some such?

Reply:
Done. Default is to run, so I've modified it.